x-pack/plugins/security_solution/public/sourcerer/experimental/components/dataview_picker/index.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DataViewPicker } from '@kbn/unified-search-plugin/public';
+import React, { useCallback, useRef, useMemo, memo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { useKibana } from '../../../../common/lib/kibana/kibana_react';
+import { DEFAULT_SECURITY_SOLUTION_DATA_VIEW_ID } from '../../constants';
+import { selectDataView } from '../../redux/actions';
+import { sourcererAdapterSelector } from '../../redux/selectors';
+
+const TRIGGER_CONFIG = {
+  label: 'Dataview',
+  color: 'danger',
+  title: 'Experimental data view picker',
+  iconType: 'beaker',
+} as const;
+
+export const DataviewPicker = memo(() => {
```

Comment:
I see _Dataview_ and _DataView_ casing around, it would be good to be consistent everywhere.
```suggestion
export const DataViewPicker = memo(() => {
```